Pivot/ViewController.swift

```diff
@@ -7,19 +7,135 @@
 //
 
 import UIKit
+import CoreMotion
+import GLKit
 
 class ViewController: UIViewController {
 
+    //MARK: Properties
+    @IBOutlet weak var rotationsTextView: UITextView!
+    @IBOutlet weak var pivotOrientationView: PivotOrientationView!
+    @IBAction func reset(_ sender: Any) {
+        total = 0
+    }
+    
+    let motionManager = CMMotionManager()       // access data from CoreMotion API
+    var timer: Timer!                           // schedules updates at regular interval from motionManager
```

Comment:
Avoid `!` whenever possible, and if you reach for it, it's a good occasion to check yourself to make sure there's not a mistake. This if fine and shouldn't create problems, but you could also easily re-write this to never be `nil`, which will be better at expressing the expectations for this object. lmk if you have questions about how you could do that.